source/connect/tls.txt

```diff
@@ -0,0 +1,267 @@
+.. _php-tls:
+
+========================================
+Configure Transport Layer Security (TLS) 
+========================================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: security, authentication, transport layer security, encrypt
+
+Overview
+--------
+
+In this guide, you can learn how to use the :wikipedia:`TLS <Transport_Layer_Security>`
+protocol to secure your connection to a MongoDB deployment.
+
+When you enable TLS for a connection, the {+driver-short+} performs the following actions:
+
+- Uses TLS to connect to the MongoDB deployment
+- Verifies the deployment's certificate
+- Ensures that the certificate certifies the deployment
+
+To learn how to configure your MongoDB deployment for TLS, see the
+:manual:`TLS configuration guide </tutorial/configure-ssl/>` in the
+{+mdb-server+} manual. 
+
+.. note::
+
+   This page assumes prior knowledge of TLS/SSL and access to valid certificates.
+   A full description of TLS/SSL, PKI (Public Key Infrastructure) certificates, and
+   Certificate Authorities (CAs) is beyond the scope of this documentation. 
+
+.. tip::
+
+   The {+driver-short+} delegates most TLS behavior to the MongoDB C Driver.
+   For information about how the C driver handles TLS, including configuration steps
+   and expected behavior, see
+   `Configuring TLS <https://www.mongodb.com/docs/languages/c/c-driver/current/libmongoc/guides/configuring_tls/#supported-libraries>`__
+   in the C driver Documentation.
+
+.. _php-enable-tls:
+
+Enable TLS
+----------
+
+To enable TLS for the connection to your MongoDB instance, set the ``tls`` connection
+option to ``true``. You can do this in two ways: by using the ``uriOptions`` parameter
+of the ``MongoDB\Client`` constructor or through a parameter in your connection string.
+
+.. include:: /includes/connect/tls-tabs.rst
+
+.. tip::
+   
+   If your connection string includes the ``+srv`` modification, which specifies the
+   SRV connection format, TLS is enabled on your connection by default.
+
+   To learn more about the SRV connection format, see
+   :manual:`SRV Connection Format </reference/connection-string/#srv-connection-format>`
+   in the {+mdb-server+} documentation.
+
+.. _php-specify-ca-file:
+
+Specify a CA File
+------------------
+
+During the TLS handshake, the MongoDB deployment presents a certificate key file to your
+application to establish its identity. Usually, a deployment's certificate has been
+signed by a well-known CA, and your application relies on this CA to validate the certificate.
```

Comment:
Q: Is CA an initialism/do we need to introduce what CA stands for before using it throughout this section?

Reply:
good question. I mention it in the note at the top of the page and specifically say "this page isn't meant to teach you what these things are"

Reply:
I don't think we need a definition for CA, but it still could be beneficial to introduce it as "certificate authority (CA)" from my point of view. Will leave it to you.